pkg/config/config.go

```diff
@@ -148,7 +148,10 @@ type RuntimeHandler struct {
 	RuntimeRoot string `toml:"runtime_root"`
 	// PrivilegedWithoutHostDevices can be used to restrict passing host devices
 	// to a container running as privileged.
-	PrivilegedWithoutHostDevices bool `toml:"privileged_without_host_devices"`
+	PrivilegedWithoutHostDevices bool `toml:"privileged_without_host_devices,omitempty"`
+	// RuntimeAllowUsernsAnnotation can be used to enable this runtime handler to interpret
+	// userns configuration in the pod's annotations
+	RuntimeAllowUsernsAnnotation bool `toml:"runtime_allow_userns_annotation,omitempty"`
```

Comment:
This option is very specific, are we planning to add other annotations later on? We may generalize it then into `allowAnnotations = ["userns"]`, etc… 

In any case I'd recommend to strip the `Runtime`/`runtime_` prefix from the option.

Reply:
that is *way* better